SP23.P03.Web/Data/SeedHelper.cs

```diff
@@ -93,6 +95,39 @@ private static async Task AddTrainStation(DataContext dataContext)
         await dataContext.SaveChangesAsync();
     }
 
+    public static async Task AddTrains(IServiceProvider serviceProvider, DataContext dataContext)
+    {
+        var trains = dataContext.Set<Train>();
+
+        if (await trains.AnyAsync())
+        {
+            return;
+        }
+
+        trains.AddRange(
+            new Train
+            {
+                Name = "Hammond Train",
+                Status = "In Use",
+                Capacity = 98
+            },
+            new Train
+            {
+                Name = "Slidell Train",
+                Status = "Out",
+                Capacity = 120
+            },
+            new Train
+            {
+                Name = "NOLA Train",
+                Status = "Repair",
+                Capacity = 111
+            });
```

Comment:
In the future there should probably be a static class that has the names for all the possible train statuses (similar to the `RoleNames` class) or Status could be made into an enum. Not necessary right now though